ldap/servers/slapd/back-ldbm/db-mdb/mdb_import.h

```diff
@@ -157,6 +157,7 @@ int dbmdb_run_ldif2db(Slapi_PBlock *pb);
 
 
 /* mdb_import_threads.c */
+const char *attr_in_list(const char *search, char **list);
```

Comment:
This line seems to be unrelated to the change (and not mentioned in the commit message). Is it here by mistake or do I miss something?